src/components/Board.js

```diff
@@ -13,10 +19,19 @@ const generateSquareComponents = (squares, onClickCallback) => {
 }
 
 const Board = ({ squares, onClickCallback }) => {
-  const squareList = generateSquareComponents(squares, onClickCallback);
-  console.log(squareList);
+  const squareList = generateSquareComponents(squares);
+
   return <div className="grid" >
-    {squareList}
+    {squareList.map(squ => {
+      return (
+        <Square
+          id={squ.id}
+          key={squ.key}
```

Comment:
this would be `squ.id` because squares do not have a key. You are creating one with their unique id.